main/peripherals/bme280.h

```diff
@@ -23,17 +24,65 @@ struct registerMapBME {
     uint8_t humidityLSB;
 };
 
+typedef struct registerCalibrationMapBME {
+    uint16_t dig_T1;
+    int16_t dig_T2;
+    int16_t dig_T3;
+    uint16_t dig_P1;
+    int16_t dig_P2;
+    int16_t dig_P3;
+    int16_t dig_P4;
+    int16_t dig_P5;
+    int16_t dig_P6;
+    int16_t dig_P7;
+    int16_t dig_P8;
+    int16_t dig_P9;
+    uint8_t dig_H1;
+    int16_t dig_H2;
+    uint8_t dig_H3;
+    int16_t dig_H4;
+    int16_t dig_H5;
+    int16_t dig_H6;
+};
+
 enum registerPointerValuesBME {
+    // Calibration Registers
+    BME280_REGISTER_DIG_T1 = 0x88,
+    BME280_REGISTER_DIG_T2 = 0x8A,
+    BME280_REGISTER_DIG_T3 = 0x8C,
+    BME280_REGISTER_DIG_P1 = 0x8E,
+    BME280_REGISTER_DIG_P2 = 0x90,
+    BME280_REGISTER_DIG_P3 = 0x92,
+    BME280_REGISTER_DIG_P4 = 0x94,
+    BME280_REGISTER_DIG_P5 = 0x96,
+    BME280_REGISTER_DIG_P6 = 0x98,
+    BME280_REGISTER_DIG_P7 = 0x9A,
+    BME280_REGISTER_DIG_P8 = 0x9C,
+    BME280_REGISTER_DIG_P9 = 0x9E,
+    BME280_REGISTER_DIG_H1 = 0xA1,
+    BME280_REGISTER_DIG_H2 = 0xE1,
+    BME280_REGISTER_DIG_H3 = 0xE3,
+
+    // Overlapping Calibration Registers
+    BME280_REGISTER_DIG_H4,
+    BME280_REGISTER_DIG_H5,
```

Comment:
Add a comment denoting looking into this at a future point (these registers overlap, are we handling this correctly?)